ucm2/sof-soundwire/sof-soundwire.conf

```diff
@@ -93,9 +93,9 @@ If.multi_mic {
 
 If.multi_headset {
 	Condition {
-		Type RegexMatch
-		Regex "${var:HeadsetCodec1}"
-		String "${var:MultiCodec1}"
+		Type String
+		Haystack "${var:HeadsetCodec1}"
+		Needle "${var:MultiCodec1}"
```

Comment:
This seems like an optimization, but `Haystack` and `Needle` should be swapped to follow the original regex match. The whole `HeadsetCodec1` string should be in `MultiCodec1'.